test/fork/utils.js

```diff
@@ -1,70 +1,133 @@
+  ENZYMEV4_VAULT_PRICE_FEED_ORACLE_AGGREGATOR: '0xCc72039A141c6e34a779eF93AEF5eB4C82A893c7',
 };
 
 const ratioScale = toBN('10000');
 
-async function submitGovernanceProposal(categoryId, actionData, members, gv) {
+async function submitGovernanceProposal(categoryId, actionData, signers, gv) {
   const id = await gv.getProposalLength();
-  console.log(`Creating proposal ${id}`);
 
-  const from = members[0];
-  await gv.createProposal('', '', '', 0, { from });
-  await gv.categorizeProposal(id, categoryId, 0, { from });
-  await gv.submitProposalWithSolution(id, '', actionData, { from });
+  console.log(`Proposal ${id}`);
+
+  await gv.connect(signers[0]).createProposal('', '', '', 0);
+  await gv.connect(signers[0]).categorizeProposal(id, categoryId, 0);
+  await gv.connect(signers[0]).submitProposalWithSolution(id, '', actionData);
 
-  for (let i = 0; i < 3; i++) {
-    await gv.submitVote(id, 1, { from: members[i] });
+  for (let i = 0; i < signers.length; i++) {
+    await gv.connect(signers[i]).submitVote(id, 1);
   }
 
-  await time.increase(604800);
-  const closeTx = await gv.closeProposal(id, { from });
-  expectEvent(closeTx, 'ActionSuccess', { proposalId: id });
+  const tx = await gv.closeProposal(id, { gasLimit: 21e6 });
+  const receipt = await tx.wait();
+
+  assert.equal(
+    receipt.events.some(x => x.event === 'ActionSuccess' && x.address === gv.address),
+    true,
+    'ActionSuccess was expected',
+  );
 
   const proposal = await gv.proposal(id);
-  assert.equal(proposal[2].toNumber(), 3);
+  assert.equal(proposal[2].toNumber(), 3, 'Proposal Status != ACCEPTED');
 }
 
-async function submitMemberVoteGovernanceProposal(categoryId, actionData, members, gv) {
+async function submitMemberVoteGovernanceProposal(categoryId, actionData, signers, gv) {
   const id = await gv.getProposalLength();
-  console.log(`Creating proposal ${id}`);
+  await evm.connect(ethers.provider);
 
-  const from = members[0];
-  await gv.createProposal('', '', '', 0, { from });
-  await gv.categorizeProposal(id, categoryId, 0, { from });
-  await gv.submitProposalWithSolution(id, '', actionData, { from });
+  console.log(`Proposal ${id}`);
 
-  for (let i = 0; i < 3; i++) {
-    await gv.submitVote(id, 1, { from: members[i] });
+  await gv.connect(signers[0]).createProposal('', '', '', 0);
+  await gv.connect(signers[0]).categorizeProposal(id, categoryId, 0);
+  await gv.connect(signers[0]).submitProposalWithSolution(id, '', actionData);
+
+  for (let i = 0; i < signers.length; i++) {
+    await gv.connect(signers[i]).submitVote(id, 1);
   }
 
-  await time.increase(604800);
-  const closeTx = await gv.closeProposal(id, { from });
-  expectEvent(closeTx, 'ProposalAccepted', { proposalId: id });
+  await evm.increaseTime(7 * 24 * 3600); // for DMCI it needs time to pass or to have over 8k voters
+
+  const tx = await gv.closeProposal(id, { gasLimit: 21e6 });
+  const receipt = await tx.wait();
+
+  await evm.increaseTime(24 * 3600);
```

Comment:
does this mint a new block?

Reply:
This adds time, so the proposal can be closed if there are not enough votes.
https://github.com/NexusMutual/smart-contracts/blob/f552a387bb73179c125fdcf0476d7ad426d829e3/contracts/modules/governance/Governance.sol#L712